projects/kit/components/multi-select/hide-selected.pipe.ts

```diff
@@ -6,6 +6,7 @@ import {TuiMultiSelectComponent} from './multi-select.component';
 
 @Pipe({
     name: `tuiHideSelected`,
+    // eslint-disable-next-line @angular-eslint/no-pipe-impure
```

Comment:
Let's just disable this rule. Unlike `ChangeDetectionStrategy.Default`, making a pipe impure requires an explicit action by the developer which means they mean it.

Reply:
fixed